src/node/node.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FC } from "react";
+import type { INodeProps } from "./types";
+import { Elements } from "../constants";
```

Comment:
Please be consistant with quotes, use either single or double quotes throughout the code